internal/pkg/pidfile/pidfile.go

```diff
@@ -0,0 +1,84 @@
+package pidfile
+
```

Comment:
I wonder if we want to leverage this for the Warewulf daemon as well.

Reply:
Yes, I will put that on my TODO list